src/ble/CHIPBleServiceData.h

```diff
@@ -64,6 +65,20 @@ struct ChipBLEDeviceIdentificationInfo
 
     void SetProductId(uint16_t productId) { chip::Encoding::LittleEndian::Put16(DeviceProductId, productId); }
 
+    uint16_t GetAdvertisementVersion() const
+    {
+        return chip::Encoding::LittleEndian::Get16(DeviceDiscriminator) & kAdvertisementVersionMask;
+    }
+
+    // Use only 4 bits to set advertisement version
+    void SetAdvertisementVersion(uint16_t advertisementVersion)
```

Comment:
IMO, we should have an actual call to this function with advertisementVersion set as 0, just where the discriminator and other parameters are set.
You can use a macro for the version so that it can be changed when it is updated.

Reply:
Set advertisement version to 0.